.gitignore

```diff
@@ -13,3 +13,11 @@
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+
+# vim files
+*.swp
+*.swo
+
+# Temp files
+
+tmp/
```

Comment:
nit: if you're looking to exclude only the `tmp` directory at the top level of the git repo, then you can use `/tmp/` instead. Using `tmp/` will match a `tmp` directory anywhere in the repo (e.g. `foo/bar/tmp`).